File: src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.VisualBasic.Testing.MSTest/Verifier`1.vb
' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.

Imports Microsoft.CodeAnalysis.Diagnostics
Imports Microsoft.CodeAnalysis.Testing
Imports Microsoft.CodeAnalysis.Testing.Verifiers

Public Class Verifier(Of TAnalyzer As {DiagnosticAnalyzer, New})
    Inherits VisualBasicVerifier(Of TAnalyzer, EmptyCodeFixProvider, MSTestVerifier)
End Class

File: src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CSharp.Testing.MSTest/Verifier`1.cs
﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.

using Microsoft.CodeAnalysis.Diagnostics;
using Microsoft.CodeAnalysis.Testing;
using Microsoft.CodeAnalysis.Testing.Verifiers;

namespace Microsoft.CodeAnalysis.CSharp.Testing.MSTest
{
    public class Verifier<TAnalyzer> : CSharpVerifier<TAnalyzer, EmptyCodeFixProvider, MSTestVerifier>
        where TAnalyzer : DiagnosticAnalyzer, new()
    {
    }
}
